sdk/include/reactos/libs/zlib/zconf.h

```diff
@@ -349,6 +352,11 @@
 #    ifdef FAR
 #      undef FAR
 #    endif
+#    ifndef __REACTOS__
```

Comment:
Why is this needed?

Reply:
It was manually removed from previous release.
I've restored it so it's easier to maintain, track what is modified by REACTOS or not.
I've disabled it so it's stays disabled.
I'm not sur we should keep it disabled though. It's working fine with WIN32_LEAN_AND_MEAN.

Reply:
@tkreuzer
Does this answer your question ?

Reply:
I consider this solved.

Reply:
This ifdef guard should be used if this was kept in the original code, but it needs to be removed for ROS.
If it's removed from the original code and we don't need it either, it can just be removed entirely.

Reply:
Ok, let me rephrase.

This piece of code was included with the original ZLIB source code but it was removed by a ROS dev from our version of code. It have been NOT removed from original ZLIB source code.

The ifdef guard have been put here to restore this removed piece of code and keep it disabled while I don't know why it was removed in the first place.

My real question is: It is really required to keep it guarded ?
So far my tests have shown no issue unguarded.

Reply:
Indeed... I probably compared it with a wrong reference...